api_app/serializers.py

```diff
@@ -81,16 +83,10 @@ def get_no_of_connectors_executed(self, obj: Job):
 class JobSerializer(ObjectPermissionsAssignmentMixin, serializers.ModelSerializer):
```

Comment:
`ObjectPermissionsAssignmentMixin` and `get_permissions_map` should be moved to the 2 new serializers as well.

Reply:
Create a base serializer like `class JobCreateSerializerMixin(ObjectPermissionsAssignmentMixin)` to keep the common code like `validate`, `create`, `get_permissions_map` methods.